entry/src/main/resources/base/element/string.json

```diff
@@ -2,15 +2,15 @@
   "string": [
     {
       "name": "entry_MainAbility",
-      "value": "entry_MainAbility"
+      "value": "SizeAdjustingTextView"
     },
     {
       "name": "mainability_description",
```

Comment:
Is this string being used anywhere ? If not, please delete it. If being used, please replace the value with meaningful string